### PR TITLE
rf_bridge: add new actions

### PR DESCRIPTION
**Pull request by ianchi**
## Description:

Adds documentation for proposed new functionalities in `rf_bridge`

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#1819

## Checklist:

  - [ ] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/rf_bridge.rst [hunk 1]
```diff
@@ -106,7 +106,11 @@ Configuration options:
 ``rf_bridge.send_raw`` Action
 -----------------------------
 
-Send a raw RF code using this action in automations.
+Send a raw command to the onboard EFM8BB1 chip.
+You can see a list of available commands and format in the `Portisch Wiki <https://github.com/Portisch/RF-Bridge-EFM8BB1/wiki/Commands>`__
+
+This can be used to send raw RF codes in automation's, mainly for protocols that are not supported.
+If you have *Portisch* firmware installed, this raw codes can be obtained with the help of :ref:`rf_bridge-start_bucket_sniffing_action`
```
Comment (frenck) [inline components/rf_bridge.rst]:
```suggestion
If you have *Portisch* firmware installed, these raw codes can be obtained with the help of :ref:`rf_bridge-start_bucket_sniffing_action`
```

File: components/rf_bridge.rst [hunk 2]
```diff
@@ -267,7 +271,71 @@ Configuration options:
 
         id(rf_bridge).stop_advanced_sniffing();
 
+.. _rf_bridge-start_bucket_sniffing_action:
+
+``rf_bridge.start_bucket_sniffing`` Action
+-------------------------------------------
```
Comment (frenck) [inline components/rf_bridge.rst]:
```suggestion
``rf_bridge.start_bucket_sniffing`` Action
------------------------------------------
```

File: components/rf_bridge.rst [hunk 3]
```diff
@@ -267,7 +271,71 @@ Configuration options:
...
+
+Tell the RF Bridge to dump raw sniffing data. Useful for getting codes for unsupported protocols.
+The raw data will be available in the log and can later be used with :ref:`rf_bridge-send_raw_action` action.
+
+.. note::
+
+    A conversion from *B1* (received) raw format to *B0* (send) raw command format should be applied.
+    For this you can use the tool `BitBucket Converter <https://bbconv.hrbl.pl/>`__
```
Comment (frenck) [inline components/rf_bridge.rst]:
```suggestion
    For this, you can use the tool `BitBucket Converter <https://bbconv.hrbl.pl/>`__
```

File: components/rf_bridge.rst [hunk 4]
```diff
@@ -267,7 +271,71 @@ Configuration options:
...
+
+.. note::
+
+    There seems to be an overflow problem in Portisch firmware and after a short while the bucket sniffing stops.
```
Comment (frenck) [inline components/rf_bridge.rst]:
```suggestion
    There seems to be an overflow problem in Portisch firmware and after a short while, the bucket sniffing stops.
```

File: components/rf_bridge.rst [hunk 5]
```diff
@@ -267,7 +271,71 @@ Configuration options:
...
+    You should re-call the action to reset and start sniffing again
```
Comment (frenck) [inline components/rf_bridge.rst]:
```suggestion
    You should re-call the action to reset and start sniffing again.
```

File: components/rf_bridge.rst [hunk 6]
```diff
@@ -267,7 +271,71 @@ Configuration options:
...
+.. code-block:: yaml
+
+    on_...:
+      then:
+        - rf_bridge.beep:
+            duration: 100 
+
+Configuration options:
+
+- **duration** (**Required**, string, :ref:`templatable <config-templatable>`): beep duration in milliseconds
```
Comment (frenck) [inline components/rf_bridge.rst]:
```suggestion
- **duration** (**Required**, string, :ref:`templatable <config-templatable>`): beep duration in milliseconds.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: components/rf_bridge.rst (hunk 2 above)
Comment (ianchi) [reply components/rf_bridge.rst]: updated

File: components/rf_bridge.rst (hunk 1 above)
Comment (ianchi) [reply components/rf_bridge.rst]: updated

File: components/rf_bridge.rst (hunk 4 above)
Comment (ianchi) [reply components/rf_bridge.rst]: updated

File: components/rf_bridge.rst (hunk 3 above)
Comment (ianchi) [reply components/rf_bridge.rst]: updated

File: components/rf_bridge.rst (hunk 5 above)
Comment (ianchi) [reply components/rf_bridge.rst]: updated

File: components/rf_bridge.rst (hunk 6 above)
Comment (ianchi) [reply components/rf_bridge.rst]: updated

**Review by jesserockz — APPROVED** [review]

**Pull request merged**